torch/onnx/utils.py

```diff
@@ -209,7 +209,8 @@ def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=Fa
         torch._C._jit_pass_onnx_scalar_type_analysis(graph)
         torch._C._jit_pass_lint(graph)
 
-        torch._C._jit_pass_onnx_fold_if(graph)
+        if dynamic_axes is None or not bool(dynamic_axes):
```

Comment:
`not bool(dynamic_axes)` this looks strange.. which exact case is this guarding against?

Reply:
`not bool(dynamic_axes)` means `dynamic_axes == {}`, see [here](https://www.geeksforgeeks.org/python-check-if-dictionary-is-empty/), I prefer this to `not dynamic_axes` to make this explicitly.